README.md

```diff
@@ -2,8 +2,85 @@
 
+Tide services are responsible for the following:
+- The Sync Server polls the WordPress.org API's for themes and plugins to process and writes them to a queue.
+- The PHPCS Server reads messages from a queue and runs reports against both plugins and themes, then sends the results back to the Tide API.
+- The Lighthouse Server reads messages from a queue and runs Google Lighthouse reports against the themes only, then sends the results back to the Tide API.
+
+## Table of Contents
+@todo
+
+### Setup
+@todo
+
+### Usage
+
+### PHP_CodeSniffer (PHPCS) Server
+@todo
+
+#### WordPress Coding Standards (WPCS)
+@todo
+
+#### PHP Compatibility
+@todo
+
+### Lighthouse Server
+This section outlines the components included in the Lighthouse integration with Tide, demonstrates how the integration processes Lighthouse audits of WordPress.org themes, and provides examples of the Tide API with Lighthouse audit results.
+
+#### Status
+Lighthouse auditing of WordPress themes has been integrated by running each theme from wp-themes.com through the Lighthouse CLI, stores the full report, and includes the summary results and a link to the full report in the Tide API. This currently functions locally, XWP is working to get this running on GCP.
+
+#### Components
+The following outlines the components added to Tide in order to integrate Lighthouse in overall Tide auditing of WordPress themes.
+1. Docker Container
+   - Lighthouse CLI
+   - Lighthouse Audit Server binary (lh-server)
+   **Note:** Uses an Alpine Image with working Chromium version. Produces consistent results with Lighthouse Chrome extension. Allows reduced image size of 432MB.
+2. Go Lighthouse Server Source Code
+   - `cmd/lh-server` for binary
+   - `src` for packages
+   - `vendor` for imported packages
+     - [github.com/aws/aws-sdk-go](https://github.com/aws/aws-sdk-go)
+     - [github.com/hhatto/gocloc](https://github.com/hhatto/gocloc)
+   - Running Tide cluster
+
+#### Process
+The following demonstrates how a WordPress theme is run through a Lighthouse audit and has its results stored and returned via the Tide API.
+1. `tide-cluster`
+   - Starts all Tide services and listens...
+@todo add screenshot
+2. `lh-server`
+   - authenticate with Tide API    
+@todo add screenshot
+   - polls a job queue (SQS) for messages to process
+@todo add screenshot
+   - downloads theme and calculates checksum
+@todo add screenshot
+   - runs source through `gocloc` to get code information
+   - scans source for Theme header information (required for Lighthouse Audit)
+@todo add screenshot
+   - runs theme through `lighthouse-cli` at `https://wp-themes.com/<theme-slug>` and keeps polling for next job.
+@todo add screenshot
+   - saves full report to a file store (S3)
+@todo add screenshot
```

Comment:
Missing a step where it takes the full report and generates the detailed report.

Reply:
@valendesigns @rheinardkorf are there any commands/screenshots that help highlight that added step?